utils/logger.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import "k8s.io/klog"
+
+var (
+	DebugMode = true
```

Comment:
Resolve in the next PR:

"Debug mode" is given inherently through V=3 or above. That's a better practice, to plan logging based on verbosity.

The `Debug` function below practically adds no information to the logging.